src/common/__tests__/hotbar-store.test.ts

```diff
@@ -84,69 +36,81 @@ function getMockCatalogEntity(data: Partial<CatalogEntityData> & CatalogEntityKi
   } as CatalogEntity;
 }
 
-const testCluster = getMockCatalogEntity({
-  apiVersion: "v1",
-  kind: "Cluster",
-  status: {
-    phase: "Running",
-  },
-  metadata: {
-    uid: "test",
-    name: "test",
-    labels: {},
-  },
-});
-
-const minikubeCluster = getMockCatalogEntity({
-  apiVersion: "v1",
-  kind: "Cluster",
-  status: {
-    phase: "Running",
-  },
-  metadata: {
-    uid: "minikube",
-    name: "minikube",
-    labels: {},
-  },
-});
-
-const awsCluster = getMockCatalogEntity({
-  apiVersion: "v1",
-  kind: "Cluster",
-  status: {
-    phase: "Running",
-  },
-  metadata: {
-    uid: "aws",
-    name: "aws",
-    labels: {},
-  },
-});
-
 describe("HotbarStore", () => {
   let di: DiContainer;
   let hotbarStore: HotbarStore;
+  let testCluster: CatalogEntity;
+  let minikubeCluster: CatalogEntity;
+  let awsCluster: CatalogEntity;
+  let logger: jest.Mocked<Logger>;
 
   beforeEach(async () => {
-    di = getDiForUnitTesting({ doGeneralOverrides: true });
+    di = getDiForUnitTesting({ doGeneralOverrides: true, overrideHotbarStore: false });
```

Comment:
There's `di.unoverride(hotbarStoreInjectable)` to support legacy unit tests with side-effects. Unoverride is preferrable over complication of `getDiForUnitTesting`.

Reply:
Thanks for the info. I did not know about that.